aspnetcore/mvc/advanced/custom-model-binding.md

```diff
@@ -42,7 +42,8 @@ public IModelBinder GetBinder(ModelBinderProviderContext context)
 
     if (context.Metadata.ModelType == typeof(byte[]))
     {
-        return new ByteArrayModelBinder();
+        var loggerFactory = context.Services.GetRequiredService<ILoggerFactory>();
+        return new ByteArrayModelBinder(loggerFactory);
```

Comment:
This was slightly different to the source it's attempting to show, so I synced it up for completeness.